_scripts/install_all.sh

```diff
@@ -59,6 +65,6 @@ docker pull pafortin/goaws
 
 docker pull redis
 
-docker pull mysql/mysql-server:5.6
+docker pull mysql/mysql-server:5.7
```

Comment:
fxa uses 5.6 https://github.com/mozilla/fxa-auth-db-mysql#prerequisites what do we do 😱

Reply:
Can we have both running?